scenes/TestPlanet.tscn

```diff
@@ -0,0 +1,29 @@
+[gd_scene load_steps=5 format=2]
```

Comment:
I will delete this when merging because we really don't need this at this point

Reply:
would you please delete this file because it won't work. (we talked about lately)